### ISSUE TITLE
Improve contrast ratio

### ISSUE THREAD

**Issue by coogie** [coogie/whatismyviewport#26]
As per [the Lighthouse report][0], WIMVP has two nodes failing on A11Y contrast ratio.

Lighthouse reports are run on the Light theme variant, and so this message is in relation to that. However, the author should ensure that A11Y contrast ratios are sufficient in Dark theme also.

[0]: https://lighthouse-dot-webdotdevsite.appspot.com/lh/html?url=https://whatismyviewport.com/
